governance.md

```diff
@@ -1,98 +1,137 @@
+[dashboard on tekton.devstats.cd.foundation](https://tekton.devstats.cd.foundation/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All)
+will show GitHub based contributions that are > 20 total.
+We expect to increase the contribution count required going forward.
 
-The transition process MUST:
+### Candidate Eligibility
 
-* Organize, execute, and validate an election for replacing bootstrap members (they may re-run, but would need to be re-elected in order to stay)
-* Define the term lengths for newly-elected individuals, ideally so not all members change out at once
-* Provide documentation for the community and committee members sufficient to smoothly continue the established practices of the committee
+Candidates themselves must be contributors who are eligible to vote, and must
+be nominated by contributors from at least two companies, which can include
+their own, and they can self-nominate.
+Any nominee who accepts the nomination will be on the ballot.
 
-## Contribution Process
+### Election Process
 
-The committee MUST define a contribution process that:
+Elections will be held using time-limited [Condorcet](https://en.wikipedia.org/wiki/Condorcet_method)
+ranking on [CIVS](http://civs.cs.cornell.edu/) using the [Schulze method](https://en.wikipedia.org/wiki/Schulze_method).
+The top vote getters will be elected to the open seats. This is the same process
+used by the Kubernetes project.
 
-* Explains to potential contributors how/if they can add code to the repository/repositories
-* Documents Workflow and management of pull requests
-* Identifies who is authorized to commit or revert code
-* Identifies automation is required for normal operations
-* Defines how release decisions are made
-  * Who is authorized to release and when.
-  * Frequency limits
-* Defines the documentation process
-* Defines what CLA process is required and how it is programmatically enforced before code is merged
+Details about the schedule and logistics of the election will be announced in a
+timely manner by the election officers to elegible candidates and voters via the
+tekton-dev@goooglegroups.com mailing list.
 
-## Code of Conduct
+### Election Officers
 
-The code of conduct MUST set expectations for contributors on expected behavior, as well as explaining the consequences of violating the terms of the code.
-The [Contributor Covenant](https://www.contributor-covenant.org) has become the de facto standard for this language.
+For every election, the governance board wll choose three election officers,
+by the following criteria, so as to promote healthy rotation and diversity:
 
-Members of the governance committee will be responsible for handling [Tekton code of conduct](code-of-conduct.md)
-violations via tekton-code-of-conduct@googlegroups.com.
+* election officers must be eligible to vote
+* two election officers should have served before
```

Comment:
i think this one might be impossible, i think i did this on my own last time? XD

Reply:
Thanks, added a comment